internal/clients/team4/judge.go

```diff
@@ -108,29 +108,45 @@ func (j *judge) HistoricalRetributionEnabled() bool {
 // CallPresidentElection is called by the judiciary to decide on power-transfer
 // COMPULSORY: decide when to call an election following relevant rulesInPlay if you wish
 func (j *judge) CallPresidentElection(monitoring shared.MonitorResult, turnsInPower int, allIslands []shared.ClientID) shared.ElectionSettings {
-	// example implementation calls an election if monitoring was performed and the result was negative
-	// or if the number of turnsInPower exceeds 3
+
 	var electionsettings = shared.ElectionSettings{
 		VotingMethod:  shared.Runoff,
```

Comment:
Why is this hardcoded?

Reply:
is that just a question on the comment? it was hardcoded in baseclient implementation, but is not anymore, as it's rule-driven

Reply:
is that just a question on the comment? it was hardcoded in baseclient implementation, but is not anymore, as it's rule-driven

Reply:
Sorry, VotingMethod:  shared.Runoff

Reply:
do we have a better way of deciding whether it should be runoff, instant runoff or the 3rd option?

Reply:
I could do random, but I'm not sure that's better

Reply:
Do we not need to use the method that's chosen in the rules?

Reply:
there isn't a rule on voting method